src/auth-service/controllers/create-user.js

```diff
@@ -155,12 +156,15 @@ const createUser = {
       const { email, phoneNumber, uid, providerId, providerUid } = req.body;
       const hasErrors = !validationResult(req).isEmpty();
       if (hasErrors) {
+        logObject("hasErrors", hasErrors);
         let nestedErrors = validationResult(req).errors[0].nestedErrors;
+
         logger.error(
           `input validation errors ${JSON.stringify(
-            manipulateArraysUtil.convertErrorArrayToObject(nestedErrors)
```

Comment:
the runtime issue which was causing the gateway timeouts (with bad input) whenever user was looking up users from Firebase